BridgeCareApp/BridgeCare/DataAccessLayer/InvestmentStrategies.cs

```diff
@@ -41,16 +41,12 @@ public IQueryable<InvestmentStrategyModel> GetInvestmentStrategies(int simulatio
                         BudgetOrder = p.INVESTMENTS.BUDGETORDER,
                         YearlyBudgets = p.YEARLYINVESTMENTs.Select(m => new InvestmentStrategyYearlyBudgetModel
                         {
+                            Id = m.YEARID,
                             Year = m.YEAR_,
-                            Budget = p.YEARLYINVESTMENTs
-                            .Where(n => n.YEAR_ == m.YEAR_)
-                            .Select(f => new InvestmentStrategyBudgetModel()
-                            {
-                                budgetAmount = f.AMOUNT,
-                                budgetName = f.BUDGETNAME
-                            }).ToList()
+                            BudgetAmount = m.AMOUNT,
+                            BudgetName = m.BUDGETNAME
                         }).ToList()
-                    }).ToList();
+                    }).Where((t) => t.SimulationId == simulationId).ToList();
```

Comment:
curious if this does anything or is it redundant

Reply:
This is the filter to fetch only one item, instead of the list on items.